contents/tutorials/vue-feature-flags.md

```diff
@@ -0,0 +1,195 @@
+      "<ph_project_api_key>",
+      {
+        api_host: "<ph_instance_address>",
+      }
+    );
+  },
+};
+```
+
+Replace `<ph_project_api_key>` and `<ph_instance_address>` with your your PostHog API key and host. You can find these in your [project settings](https://us.posthog.com/settings/project).
+
+Finally, activate your plugin in `main.js`:
+
+```js file=main.js
+import { createApp } from 'vue'
+import App from './App.vue'
+import posthogPlugin from '../plugins/posthog';
+
+const app = createApp(App);
+app.use(posthogPlugin);
+app.mount('#app')
+```
+
+Once you’ve done this, reload your app. You should begin seeing events in the [PostHog events explorer](https://us.posthog.com/events).
+
+<ProductScreenshot
+  imageLight={EventsLight} 
+  imageDark={EventsDark} 
+  alt="Feature flag created in PostHog" 
+  classes="rounded"
+/>
+
+## Creating a feature flag
+
+With PostHog set up, your app is ready for feature flags. To create one, go to the [feature flags tab](https://us.posthog.com/feature_flags) in PostHog and click "New feature flag." Enter a flag key (like `my-cool-flag`), set the release condition to roll out to 100% of users, and press "Save."
+
+<ProductScreenshot
+  imageLight={CreateFlagLight} 
+  imageDark={CreateFlagDark} 
+  alt="Feature flag created in PostHog" 
+  classes="rounded"
+/>
+
+You can customize your [release conditions](/docs/feature-flags/creating-feature-flags#release-conditions) with rollout percentages, and [user](/docs/product-analytics/user-properties) or [group properties](/docs/product-analytics/group-analytics) to fit your needs.
+
+## Implementing our flag code
+
+Once created, we can add our feature flag to our app. We do this using the `this.$posthog.onFeatureFlags` callback.
+
+In our example, we fetch our feature flag and update the text on the page based on its value:
+
+```vue file=App.vue
+<template>
+  <div id="app">
+    <h1>{{ titleText }}</h1>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'App',
+  data() {
+    return {
+      titleText: 'Waiting for feature flag'
+    };
+  },
+  mounted() {
+    const posthog = this.$posthog;
+    posthog.onFeatureFlags(() => {
+      if (posthog.isFeatureEnabled('my-cool-flag')) {
+        this.titleText = 'Our flag is enabled!';
+      } else {
+        this.titleText = 'Our flag is disabled!';
+      }
+    });
+  },
+}
+</script>
+```
+
+When you run your app now, you should see the updated text.
+
+![New app after adding the flag](../images/tutorials/vue-feature-flags/flag-enabled-in-app.png)
+
+### Bootstrapping feature flags
+
+You may notice flickering when you load your app. This is because PostHog is making an asynchronous request to fetch the flag value and so the text is only updated when the request returns. To have your feature flags available immediately, you can initialize PostHog with precomputed values until PostHog has had a chance to fetch them. This is called [bootstrapping](/docs/feature-flags/bootstrapping).
```

Comment:
I think this paragraph could be simplied, one sentence less probably. 
```suggestion
You may notice the text flickers when you load your app. This is because PostHog is making an asynchronous request to fetch the flag value and the text only updates when the request returns. To prevent this and have your feature flags available immediately, you can initialize PostHog with precomputed flag values until it has a chance to fetch them. This is called [bootstrapping](/docs/feature-flags/bootstrapping).
```